app/models/journeyDomain/transportMeans/TransportMeansActiveDomain.scala

```diff
@@ -39,26 +40,32 @@ case class TransportMeansActiveDomain(
   nationality: Option[Nationality],
   customsOffice: CustomsOffice,
   conveyanceReferenceNumber: Option[String]
-)(index: Index)
+)(index: Index)(implicit phaseConfig: PhaseConfig)
     extends JourneyDomainModel {
 
   def asString(implicit messages: Messages): String =
     TransportMeansActiveDomain.asString(identification, identificationNumber)
 
-  override def routeIfCompleted(userAnswers: UserAnswers, mode: Mode, stage: Stage): Option[Call] = Some(
-    userAnswers.get(OfficesOfTransitSection) match {
-      case Some(_) => activeRoutes.CheckYourAnswersController.onPageLoad(userAnswers.lrn, mode, index)
-      case None    => transportMeansRoutes.TransportMeansCheckYourAnswersController.onPageLoad(userAnswers.lrn, mode)
+  override def routeIfCompleted(userAnswers: UserAnswers, mode: Mode, stage: Stage): Option[Call] =
```

Comment:
How big a change would it be to have `def routeIfCompleted(userAnswers: UserAnswers, mode: Mode, stage: Stage)(implicit phaseConfig: PhaseConfig)` ?